src/bot.py

```
@@ -31,23 +33,35 @@
     try:
+        completion_start_time = time.perf_counter()
         completion = openai.chat.completions.create(
             model=MODEL_NAME,
             messages=messages,
         )
+        completion_end_time = time.perf_counter()
+        completion_duration = completion_end_time - completion_start_time
+        logger.info(f"Got completion from LLM in {completion_duration:.2f} seconds")
         return completion.choices[0].message.content
+
     except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)